src/Components/Message/Alert.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<div class="c-application c-alert_container" :class="classes">
```

Comment:
```suggestion
	<div class="c-application c-alert--container" :class="classes">
```
`_`로 되어있는 클래스명들을 `--`로 바꿔야겠네요